cf-java-logging-support-core/src/main/java/com/sap/hcp/cf/logging/common/converter/DefaultCustomFieldsConverter.java

```diff
@@ -107,7 +107,7 @@ private void addCustomFields(ArrayComposer<ObjectComposer<JSONComposer<String>>>
 			// Let argument CustomField take precedence over MDC
 			CustomField field = customFields.get(key);
 			if (field != null) {
-				value = field.getValue();
+                value = String.valueOf(field.getValue());
```

Comment:
This class will be deleted, when the change is also implemented for log4j2. Then this indentation will no longer matter.